pages/receive/Receive.tsx

```diff
@@ -77,92 +35,11 @@ export function Receive() {
     });
   }
 
-  // TODO: move this somewhere else to have app-wide notifications of incoming payments
```

Comment:
Maybe we can address this to show a toast whenever we get a payment? because received payments via lightning address will not show any more - we are removing a feature here.

Reply:
Then we can make this provider which keeps pushing notifications via toast in-app (everywhere, not just receive), will add a commit